proto/asn1-pdu/asn1_universal_types_to_der.h

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+#ifndef PROTO_ASN1_PDU_UNIVERSAL_TYPES_TO_DER_H_
+#define PROTO_ASN1_PDU_UNIVERSAL_TYPES_TO_DER_H_
+
+#include <stdint.h>
+
+#include <vector>
+#include <algorithm>
```

Comment:
No need for `<algorithm>` here, or for `common.h` - or at least, those belong to the `.cc` files as appropriate :)